### PR TITLE
Remove space between buttons v3

### LINKED ISSUE #5555: Move 'Set status' dropdown and 'Save' button together
**Is your feature request related to a problem? Please describe.**
The 'Set status' and 'Save' buttons are on opposite sides of the Header. This placement can be confusing for users becuase they expect all actions to be grouped in the same area.

**Describe the solution you'd like**
The 'Set status' dropdown should be moved to the left side of the Header so it is next to the 'Delete unpublished changes' button.

**Describe alternatives you've considered**
None

**Additional context**

**Figure: Similar UI elements on opposite sides of the header**

**Figure: Mockup - Proposed button positions**

See this YouTube video that goes through this as well as some related changes - https://youtu.be/8vhHKf0nCS4

### PR DESCRIPTION
**Pull request by taineriley1**
**Summary**
Closes #5555
This PR attempts to implement the same changes as #5585 and #5685

I have moved the dropdowns from the right of the toolbar to the left.
I have also moved the 'delete' button to always be on the end.

**Test plan**
Tested locally in simple and editorial workflow.
Yarn test and cypress tests pass.

**Checklist**

Please add a `x` inside each checkbox:

- [x] I have read the [contribution guidelines](../CONTRIBUTING.md).
- [x] Code is formatted via running `yarn format`.
- [x] Tests are passing via running `yarn test`.
- [ ] The status checks are successful (continuous integration). Those can be seen below.

**A picture of a cute animal (not mandatory but encouraged)**
image

### CODE REVIEW

**Comment by taineriley1** [comment]
The failing build seems to be a network error.
@erezrokah can you please investigate?

File: packages/netlify-cms-core/src/components/Editor/EditorToolbar.js [hunk 1]
```diff
@@ -546,18 +553,11 @@ export class EditorToolbar extends React.Component {
...
 
-    const canCreate = collection.get('create');
     if (!isNewEntry && !hasChanged) {
-      return (
-        <>
-          {this.renderDeployPreviewControls(t('editor.editorToolbar.deployButtonLabel'))}
-          {this.renderExistingEntrySimplePublishControls({ canCreate })}
-        </>
-      );
+      return <>{this.renderDeployPreviewControls(t('editor.editorToolbar.deployButtonLabel'))}</>;
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorToolbar.js:559]: Can we remove the fragment? Seems like it's redundant

File: packages/netlify-cms-core/src/components/Editor/EditorToolbar.js [hunk 2]
```diff
@@ -605,32 +619,19 @@ export class EditorToolbar extends React.Component {
...
+    const { currentStatus, isNewEntry, t } = this.props;
 
     if (currentStatus) {
       return (
-        <>
-          {this.renderDeployPreviewControls(t('editor.editorToolbar.deployPreviewButtonLabel'))}
-          {this.renderWorkflowStatusControls()}
-          {this.renderNewEntryWorkflowPublishControls({ canCreate, canPublish })}
-        </>
+        <>{this.renderDeployPreviewControls(t('editor.editorToolbar.deployPreviewButtonLabel'))}</>
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorToolbar.js:626]: Same comment about removing the fragment

File: packages/netlify-cms-core/src/components/Editor/EditorToolbar.js [hunk 3]
```diff
@@ -605,32 +619,19 @@ export class EditorToolbar extends React.Component {
...
      * Publish control for published workflow entry.
      */
     if (!isNewEntry) {
-      return (
-        <>
-          {this.renderDeployPreviewControls(t('editor.editorToolbar.deployButtonLabel'))}
-          {this.renderExistingEntryWorkflowPublishControls({ canCreate, canPublish, canDelete })}
-        </>
-      );
+      return <>{this.renderDeployPreviewControls(t('editor.editorToolbar.deployButtonLabel'))}</>;
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorToolbar.js:634]: I think we can remove this fragment too

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks @bradystroud!

I haven't tested it yet, but added a few comments on the code

**Comment by taineriley1** [comment]
Thanks @erezrokah, removed the unnecessary fragments in the latest commit.

File: packages/netlify-cms-core/src/components/Editor/EditorToolbar.js [hunk 4]
```diff
@@ -302,13 +302,20 @@ export class EditorToolbar extends React.Component {
   }
 
   renderSimpleSaveControls = () => {
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorToolbar.js:304]: Should we rename this function to `renderSimpleControls`

File: packages/netlify-cms-core/src/components/Editor/EditorToolbar.js [hunk 5]
```diff
@@ -546,18 +553,11 @@ export class EditorToolbar extends React.Component {
   };
 
   renderSimplePublishControls = () => {
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorToolbar.js:555]: Should we rename this function to `renderSimpleDeployPreviewControls`?

File: packages/netlify-cms-core/src/components/Editor/EditorToolbar.js [hunk 6]
```diff
@@ -605,32 +619,17 @@ export class EditorToolbar extends React.Component {
   };
 
   renderWorkflowPublishControls = () => {
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorToolbar.js:621]: Same rename comment, maybe `renderWorkflowDeployPreviewControls`?

File: packages/netlify-cms-core/src/components/Editor/EditorToolbar.js [hunk 7]
```diff
@@ -546,18 +553,11 @@ export class EditorToolbar extends React.Component {
...
-          {this.renderDeployPreviewControls(t('editor.editorToolbar.deployButtonLabel'))}
-          {this.renderExistingEntrySimplePublishControls({ canCreate })}
-        </>
-      );
+      return this.renderDeployPreviewControls(t('editor.editorToolbar.deployButtonLabel'));
     }
-    return this.renderNewEntrySimplePublishControls({ canCreate });
   };
 
   renderWorkflowSaveControls = () => {
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorToolbar.js:563]: Same rename comment, maybe `renderWorkflowControls`?

File: packages/netlify-cms-core/src/components/Editor/EditorToolbar.js [hunk 8]
```diff
@@ -593,6 +599,14 @@ export class EditorToolbar extends React.Component {
       >
         {isPersisting ? t('editor.editorToolbar.saving') : t('editor.editorToolbar.save')}
       </SaveButton>,
+      currentStatus
+        ? [
+            this.renderWorkflowStatusControls(),
+            this.renderNewEntryWorkflowPublishControls({ canCreate, canPublish }),
+          ]
+        : !isNewEntry
+        ? this.renderExistingEntryWorkflowPublishControls({ canCreate, canPublish, canDelete })
```
Comment (erezrokah) [inline packages/netlify-cms-core/src/components/Editor/EditorToolbar.js:608]: This is quite hard to follow.

Do you think we can replace the second ternary operator with `!isNewEntry && this.renderExistingEntryWorkflowPublishControls`?

**Review by erezrokah — CHANGES_REQUESTED** [review]
Hi @taineriley1 and sorry for the late review.

Added a few comments.

Also, during my tests I noticed the `Save` notification hides the `delete` button.
I'll check if we can move it a bit down.
